spec/builtin-printf.test.sh

```diff
@@ -1461,3 +1461,26 @@ octal 5
 55
 37
 ## END
+
```

Comment:
Can you add a test case for +42 +077 and +0xAB  ?  And then recognize +

Now that we know it's there